### PR TITLE
Add "chrome_android" as browser name

### PR DESCRIPTION
**Pull request by WeizhongX**
Add chrome_android as a browser name in wpt.fyi. We plan to publish wpt test results for chrome android to wpt.fyi.

### CODE REVIEW

**Comment by WeizhongX** [comment]
@KyleJu I finally am able to figure out what is wrong, and made a fix. Can you pls review and approve this change?

Thanks!

File: shared/browsers.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ var defaultBrowsers = []string{
 
 // An extra list of known browsers.
 var extraBrowsers = []string{
-	"android_webview", "deno", "epiphany", "flow", "servo", "uc", "webkitgtk",
+	"chrome_android", "android_webview", "deno", "epiphany", "flow", "servo", "uc", "webkitgtk",
```
Comment (KyleJu) [inline shared/browsers.go:17]: Should be alphabetical

File: webapp/components/product-info.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ const DisplayNames = (() => {
   ['edge', 'edge-experimental'].forEach(n => m.set(n, 'Edge'));
   ['firefox', 'firefox-experimental'].forEach(n => m.set(n, 'Firefox'));
   ['safari', 'safari-experimental'].forEach(n => m.set(n, 'Safari'));
+  m.set('chrome_android', 'ChromeAndroid');
```
Comment (KyleJu) [inline webapp/components/product-info.js:12]: Put this after `m.set('android_webview', 'WebView');`

File: webapp/components/product-info.js [hunk 3]
```diff
@@ -40,7 +41,7 @@ const versionPatterns = Object.freeze({
 });
 
 // The set of all browsers known to the wpt.fyi UI.
-const AllBrowserNames = Object.freeze(['chrome', 'deno', 'edge', 'firefox', 'flow', 'safari', 'servo', 'webkitgtk', 'android_webview']);
+const AllBrowserNames = Object.freeze(['chrome_android', 'chrome', 'deno', 'edge', 'firefox', 'flow', 'safari', 'servo', 'webkitgtk', 'android_webview']);
```
Comment (KyleJu) [inline webapp/components/product-info.js:44]: put this after `chrome`

**Comment by KyleJu** [comment]
A logo should be added along with Android webview

File: shared/browsers.go (hunk 1 above)
Comment (WeizhongX) [reply shared/browsers.go:17]: sure, will update

File: webapp/components/product-info.js (hunk 2 above)
Comment (WeizhongX) [reply webapp/components/product-info.js:12]: sure, will update

File: webapp/components/product-info.js (hunk 3 above)
Comment (WeizhongX) [reply webapp/components/product-info.js:44]: We need put this one before 'chrome'. AllBrowserNames is used to create some regular expression, as chrome is a prefix of chrome_android, we need put chrome_android in front of chrome.

Comment (WeizhongX) [reply webapp/components/product-info.js:44]: I can update this to make it alphabetic order also.

**Review by KyleJu — APPROVED** [review]
LGTM

**Pull request merged**